build/cmake/modules/FindWebP.cmake

```
@@ -0,0 +1,8 @@
+find_package(WebP CONFIG)
+
+if(NOT WebP_FOUND)
+    find_package(PkgConfig)
```

We do it like this elsewhere AFAICS:
```suggestion
    find_package(PkgConfig QUIET)
```